pallets/nfts/src/benchmarking.rs

```diff
@@ -7,9 +7,11 @@ use sp_std::{boxed::Box, prelude::*};
 benchmarks! {
     create {
         let caller: T::AccountId = whitelisted_caller();
-    }: _(RawOrigin::Signed(caller), Default::default())
+        let series_id: T::NFTSeriesId = 10.into();
```

Comment:
Does this work even when benchmarking the actual runtime? Not just the mocks? You may have to do a `T::NFTSeriesId::default()`

Reply:
How do you benchmark the actual runtime?
Is the following command running the benchmarks with mocks or with the real runtime?
`cargo run --release --all-features -- benchmark --chain local --raw --wasm-execution compiled --execution wasm --pallet ternoa_nfts --extrinsic "*" --steps 100 --repeat 50000 --output .`


Btw, I am getting different read/write results depending on if the `series_id` is the default value (0) or a non zero value.

Reply:
I think your command is right. Your results may make sense given the code depends on the value of `series_id`, you have two ways to fix this:
- either make the code more similar in both cases
- alternatively, you could also use different weight functions based on the value of `series_id` (this would be optimal)